tests/core/conftest.py

```diff
@@ -219,3 +225,18 @@ async def form_bot_agent(trained_async) -> Agent:
     )
 
     return Agent.load_local_model(zipped_model)
+
+
+@pytest.fixture
```

Comment:
this shouldn't be a function scoped fixture (nor should the other examples in this file...) as it will retrain the model for every test which is just a waste of time. should rather be session scoped